### PR TITLE
Added diff script for easily comparing two package versions.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
Makes it easier to see what has changed between two package version, can either be run by the developer itself (add to guidelines) or even by ci.....

File: scripts/diff_version.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+if [ "$#" -ne 3 ]; then
+  echo "Usage: $0 package-name version1 version1 (e.g., nexus 3 4)" >&2
+  exit 1
+fi
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+REPO_DIR=$SCRIPTS_DIR/../repo/packages
+
+echo "Diffing package $1 version $2 against $3"
```
Comment (jsancio) [inline scripts/diff_version.sh]: We should say revision instead of version.

**Comment by joerg84** [comment]
@ryadav88 PTAL

**Comment by cruhland** [comment]
@joerg84 would you mind updating the README to mention this script? The [Submit your Package](https://github.com/mesosphere/universe#submit-your-package) section seems appropriate.

**Comment by ryadav88** [comment]
@cruhland we talked about this. I'll update the README once this is merged.

**Pull request merged**